rust/agama-lib/src/dbus.rs

```diff
@@ -58,7 +58,7 @@ macro_rules! property_from_dbus {
 /// NOTE: we could follow a different approach like building our own type (e.g.
 /// using the newtype idiom) and offering a better API.
 ///
-/// * `source`: hash map containing non-onwed values ([zbus::zvariant::Value]).
+/// * `source`: hash map containing non-onwed values ([enum@zbus::zvariant::Value]).
```

Comment:
I did not know about that `enum@` prefix.

Reply:
It's a disambiguation suggested by `rustdoc` because there is also a `#[derive(Value)]`